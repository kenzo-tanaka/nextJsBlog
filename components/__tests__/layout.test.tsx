import React from "react";
import renderer from "react-test-renderer";
import Layout from "../layout";

test("Layout", () => {
  const component = renderer.create(<Layout children={""} />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
