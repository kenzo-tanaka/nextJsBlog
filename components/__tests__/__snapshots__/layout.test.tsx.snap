// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout 1`] = `
<div
  className="max-w-4xl mx-auto px-2 sm:px-6 lg:px-8"
>
  <main>
    <div
      className="shadow-md bg-white"
    >
      
    </div>
  </main>
  <p
    className="mt-7 text-center"
  >
    <a
      className="text-gray-400 underline"
      href="https://github.com/kenzo-tanaka/nextJsBlog"
      target="_blank"
    >
      SourceCode
    </a>
  </p>
</div>
`;

exports[`Navbar 1`] = `
<div
  className="max-w-4xl mx-auto px-2 sm:px-6 lg:px-8"
>
  <main>
    <div
      className="shadow-md bg-white"
    >
      
    </div>
  </main>
  <p
    className="mt-7 text-center"
  >
    <a
      className="text-gray-400 underline"
      href="https://github.com/kenzo-tanaka/nextJsBlog"
      target="_blank"
    >
      SourceCode
    </a>
  </p>
</div>
`;
